src/client/context/BehandlingerContext.js

```diff
@@ -45,21 +47,21 @@ export const BehandlingerProvider = ({ children }) => {
     const fetchBehandlingsoversikt = async () => {
         setPersonTilBehandling(undefined);
         const oversikt = await fetchBehandlingsoversiktSinceYesterday();
+        setBehandlingsoversikt(oversikt);
+        setIsFetchingBehandlingsoversikt(false);
```

Comment:
Ideelt sett burde denne kalles i samme funksjon som den settes til `true`. Da er det mindre sjanse for inkonsistens i fremtiden, eksempelvis i forbindelse med refaktorisering eller at `fetchBehandlingsoversiktSinceYesterday` blir kalt fra et annet sted.

Reply:
Enig, vet ikke hvorfor jeg ikke gjorde det. Legger den i en `.finally()` på `fetchBehandlingsoversiktSinceYesterday`